Python/ibmcloudsql/SQLQuery.py

```diff
@@ -555,34 +566,15 @@ def list_cos_objects(self, url):
             object_url = object_url[:-1]
         fourth_slash = object_url.replace('/', 'X', 3).find('/')
 
-        response = requests.get(
-            object_url[:fourth_slash] + '?prefix=' + object_url[fourth_slash + 1:],
-            headers=self.request_headers,
-        )
+        cos = self._get_cos_resource(endpoint)
 
-        if response.status_code == 200 or response.status_code == 201:
-            ns = {'s3': 'http://s3.amazonaws.com/doc/2006-03-01/'}
-            responseBodyXMLroot = ET.fromstring(response.content)
-            bucket_name = responseBodyXMLroot.find('s3:Name', ns).text
-            bucket_objects = []
-            if responseBodyXMLroot.findall('s3:Contents', ns):
-                for contents in responseBodyXMLroot.findall('s3:Contents', ns):
-                    key = contents.find('s3:Key', ns)
-                    object_url = "cos://{}/{}/{}".format(endpoint, bucket_name, key.text)
-                    size = contents.find('s3:Size', ns)
-                    bucket_objects.append({'Key': object_url, 'Size': size.text})
-            else:
-                print('There are no objects in location {}'.format(url))
-                return
-        else:
-            print("Object listing for location {} failed with http code {}".format(url, response.status_code))
-            return
+        objects = cos.Bucket(bucket).objects.filter(Prefix=prefix)
+        obj_summary_columns = ['bucket_name', 'key']
 
-        objects_df = pd.DataFrame(columns=['ObjectURL', 'Size'])
-        for object in bucket_objects:
-            objects_df = objects_df.append([{'ObjectURL': object['Key'], 'Size': object['Size']}], ignore_index=True)
-
-        return objects_df
+        return pd.DataFrame(
+            [[obj['bucket_name'], obj['key']] for obj in objects],
```

Comment:
Thank you @shawnzhu 

Seems there's still a little glitch here:
```
Traceback (most recent call last):
  File "ibmcloudsql/test.py", line 155, in <module>
    objects_df = sqlClient.list_cos_objects(test_credentials.result_location)
  File "/Users/torsten@de.ibm.com/sql-query-clients/Python/ibmcloudsql/lib/python2.7/site-packages/ibmcloudsql/SQLQuery.py", line 575, in list_cos_objects
    [[obj['bucket_name'], obj['key']] for obj in objects],
TypeError: 's3.ObjectSummary' object has no attribute '__getitem__'
```

Reply:
Also increase version in setup.py please.

Reply:
fixed the glitch in https://github.com/IBM-Cloud/sql-query-clients/pull/50/commits/77bbd817d2ea076f19081162723c34005f44fe18